server/logical.sql

```diff
@@ -0,0 +1,68 @@
+DROP TABLE IF EXISTS users CASCADE; 
+DROP TABLE IF EXISTS cohort CASCADE; 
+DROP TABLE IF EXISTS events CASCADE; 
+DROP TABLE IF EXISTS recipes CASCADE; 
+DROP TABLE IF EXISTS dietary_restrictions CASCADE; 
+DROP TABLE IF EXISTS allergies CASCADE; 
+DROP TABLE IF EXISTS recipe_allergies CASCADE; 
+
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    user_name VARCHAR(60) NOT NULL,
+    user_email VARCHAR(90) NOT NULL,
+    is_admin BOOLEAN NOT NULL,
+    is_lunch_maker BOOLEAN NOT NULL,
+    is_lunch_shopper BOOLEAN NOT NULL,
+    user_location VARCHAR(120) NOT NULL,
+    cohort_id INT REFERENCES cohort(id),
+    user_password VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE cohort(
+    id SERIAL PRIMARY KEY,
+    class_number INT NOT NULL,
+    region VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE events(
+    id SERIAL PRIMARY KEY,
+    meeting_location VARCHAR(60),
```

Comment:
Feels like there could be a lot of addresses that would exceed these counts. Perhaps 256 or 1024 characters would be better here?

Reply:
Or you can have more column for the event location instead of meeting location, like Postcode (15),  address_1(50), county(50), city(50),,,and so on.